docs/core/actions.rst

```diff
@@ -144,6 +145,18 @@ There is an example of a ``SlotSet`` event
 :ref:`above <custom_action_example>`, and a full list of possible
 events in :ref:`Events <events>`.
 
+Execute Actions in Other Code
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Rasa Core will send an HTTP ``POST`` request to your server containing
+information on which action to run. Furthermore, this request will contain all
+information about the conversation.
+
+As a response to the action call from Core, you can modify the tracker,
+e.g. by setting slots and send responses back to the user.
+All of the modifications are done using events.
+There is a list of all possible event types in :ref:`events`.
+
 Default Actions
```

Comment:
@MetcalfeTom regarding https://github.com/RasaHQ/roadmap/issues/582#issuecomment-514308105 I meant that each of these headers could get a reference, e.g. 
`.. _default-actions:` so that we can both use these references in other places in the code and also link to #default-actions in links

Reply:
OK, valuable addition but it's not in the scope of this PR, so we can return to this.

Reply:
Alright, will you add it to the list?